src/librbd/migration/HttpClient.h

```diff
@@ -9,17 +9,16 @@
 #include "librbd/io/Types.h"
 #include "librbd/migration/HttpProcessorInterface.h"
 #include "librbd/migration/Types.h"
+#include <boost/asio/ssl/stream.hpp>
```

Comment:
Nit: Move this after `boost/asio/ssl/context.hpp` include so that SSL includes are together.

Reply:
Done.